tests/jax/test_fused_attn.py

```diff
@@ -717,12 +742,20 @@ def _test_forward(
         is_training,
         qkv_layout,
         bias_shape,
+        swa,
     ):
         """
         Test forward with parameterized configs
         This test is not intended to run automatically during CI as it is time-consuming
         It is kept for development and debugging
         """
+        window_size = (-1, -1)
+        if swa:
+            window_size = (s_kv // 10, 0)
+            if s_q > s_kv:
+                pytest.skip(
+                    "seqlen_q > seqlen_kv is not supported with sliding window attention in cuDNN"
+                )
```

Comment:
It would be better to put the duplicated checks and setups into `FusedAttnRunner._check_configs()` and `FusedAttnRunner._setup_inputs()`.

Reply:
Moved duplicated checks to `FusedAttnRunner._check_configs()`.